### PR TITLE
feat(release notes): Docs RN for 4-12-24

### PR DESCRIPTION
**Pull request by austin-schaefer**
Docs release notes for week ending April 12, 2024

### CODE REVIEW

File: src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+version: '4-12-2024'
+---
+
+<Callout variant="important">
+We're trying out release notes to let you know what's changing in our docs each week. This is an experiment, so [let us know via GitHub issue](https://github.com/newrelic/docs-website/issues/new/choose) if you find them helpful!
+</Callout>
+
+### New docs
+
+* General availability for browser [Session replay](/docs/browser/browser-monitoring/browser-pro-features/session-replay/), previously in limited preview. Check it out for deeper insights into user interactions on your websites.
```
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx:13]: "browser Session replay" feels like odd capitalization to me...

"browser monitoring's Session replay" feels a little less off to me

File: src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+* Docs on our new [Unreal Engine plugin](/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application/) to monitor Unreal Engine mobile apps, and updates to mobile agent API docs with Unreal Engine plugin methods.
+* New Prometheus integration for [Prometheus Node Exporter](/docs/infrastructure/prometheus-integrations/integrations-list/node-exporter-integration/), which lets you monitor Linux hardware and kernel metrics.
+
+### Major changes
+
+* Continued rewrite of on-host integration docs to make it easier to install and ensure the docs are accurate and up to date. Up this week: [Sonarqube](/docs/infrastructure/host-integrations/host-integrations-list/sonarqube-monitoring-integration) and [Windows services](/docs/infrastructure/host-integrations/host-integrations-list/windows-services-integration).
+* Added info on new mobile user journeys feature to [Crash analysis # User journeys](/docs/mobile-monitoring/mobile-monitoring-ui/crashes/crash-analysis-page/#user-journeys).
```
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx:20]: Do you really want that "#" between crash analysis and user journeys in the link text?

File: src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+* AI monitoring: Added .NET agent [configuration options](/docs/apm/agents/net-agent/configuration/net-agent-configuration/#ai_monitoring), compatibility info ([Core](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-core/#large-language-models), [Framework](/docs/apm/agents/net-agent/getting-started/net-agent-compatibility-requirements-net-framework/#large-language-models)), and [`SetLlmTokenCountingCallback()` method](/docs/apm/agents/net-agent/net-agent-api/setllmtokencountingcallback-net-agent-api/).
+
+### Minor changes
+
+* Improved definition of "wall clock" time (a frequent source of questions) in APM [Transactions page](/docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems/) doc.
+* New dashboards permissions to restrict users' ability to create live chart URLs; see [Manage dashboards](/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard/) doc for more information.
```
Comment (paperclypse) [inline src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx:26]: semicolon; fancy!

**Review by paperclypse — APPROVED** [review]
Flagged a couple minor things, otherwise looks good

File: src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx (hunk 2 above)
Comment (austin-schaefer) [reply src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx:20]: I do but no one else seems to like it. :D

File: src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx (hunk 3 above)
Comment (austin-schaefer) [reply src/content/docs/release-notes/docs-release-notes/docs-4-12-2024.mdx:26]: I'm really letting my hair down here.

**Pull request merged**